src/pages/services.js

```diff
@@ -283,15 +283,15 @@ const ServicesPage = ({ data }) => {
                 track metrics, and power useful apps.
               </div>
               <h3 className='h4'>How we’ve helped others</h3>
-              <div className='body'>City of Louisville</div>
+              <div className='body'>Centers for Medicare and Medicaid Services</div>
               <div className='body-large'>
-                Supporting local government transparency
+                Supporting government transparency
               </div>
               <div className='body'>
-                Georgia Board of Healthcare Workforce
+                We empower federal agencies like CMS to reinstate ownership of their data on open data platforms and are working toward a future in which more public data accessibility can become a reality.
```

Comment:
The link here had some issues with curly quoting, the URL and missing a closing tag - it was `<a href=“openpaymentsdata.cms.gov”> text...` but should have been`<a href="https://openpaymentsdata.cms.gov/"> text</a>...`. Having the link here also looked a bit inconsistent compared to the other services, so moved it to the "Link" bit below, so that it matched the other "How we’ve helped others" service descriptions - feel free to move it back if we don't want to do that here.

Reply:
Thanks, I think this is a good edit!